TypeCobol/Compiler/CodeElements/CodeElement.cs

```diff
@@ -1,3 +1,4 @@
+using System;
```

Comment:
Not required, no other code modification in this file ??

Reply:
This namespace is used by IEquatable.

Reply:
One "using System;" has been deleted.
https://github.com/TypeCobolTeam/TypeCobol/pull/1600/commits/8bb22a0bed7c51638872bc38edfbcb12c9e2ab34